cs251tk/toolkit/args.py

```diff
@@ -157,6 +157,8 @@ def process_args():
     parser = build_argparser()
     args = vars(parser.parse_args())
 
+    debug = args['debug']
```

Comment:
I think that the `--debug` flag should enable the printing of the debug messages, which I think is accomplished by something like this:

```python
logging.basicConfig(level=logging.DEBUG)
```